.github/workflows/actions.yml

```diff
@@ -0,0 +1,69 @@
+name: Clerk work
+
+on:
+    schedule:  # Daily schedule.
+        - cron: 0 0 * * *
+
+    workflow_dispatch: {}  # For manual switch in actions page.
+
+    push:  # Run on update to py files or this file on the main branch.
+        branches:
+            - main
+        paths:
+            - "**.py"
+            - ".github/workflows/action.yml"
+# End of on.
+
+
+jobs:
+    scheduled:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout
+                uses: actions/checkout@v3
+
+            - name: Setup Python
+                uses: actions/setup-python@v4
+
+            # https://stackoverflow.com/a/62639424
+            - name: Cache virtual environment
+                uses: actions/cache@v3
+                id: cache-venv
```

Comment:
rather than this, can we simply cache on the `requirements.txt` file?

Reply:
On every run it will download all the dependencies, which is a waste of time. This caches those dependencies.

`requirements.txt` is already in the repo statically, so caching it doesn't make sense.

Reply:
Actually, looking at it again, since we run only once a day, even a minute extra won't really matter.

Maybe I should remove this.

Reply:
What I mean is that the cache key can be determined by the hash of `requirements.txt` so that if that file changes, GitHub will know to re-fetch. Actually, I think this might be default behavior:

https://github.com/actions/setup-python#caching-packages-dependencies

Reply:
Oh, I didn't know they had now added support for pip cache natively (https://github.com/actions/setup-python/pull/266). This was added in v3.